src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+// eslint-disable-next-line
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Kindly make the App.js component class-based this is a requirement for this milestone 👍🏿